### PR TITLE
feat(tabs): remove shim mapping and fallbacks

### PR DESCRIPTION
**Pull request by mivaylo**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number:  CDE-1234

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

### CODE REVIEW

**Comment by kevinbuhmann** [comment]
Please change the PR title to something that describes the change for consumers. What should the changelog say?

**Comment by mivaylo** [comment]
> Please change the PR title to something that describes the change for consumers. What should the changelog say?

@kevinbuhmann , pushing seems to update the PR title with the latest commit title, I will change the title in the end after I've fixed the visual snapshots diff

**Comment by kevinbuhmann** [comment]
> pushing seems to update the PR title with the latest commit title, I will change the title in the end after I've fixed the visual snapshots diff

It doesn't, but it's fine to update at the end.

**Comment by mivaylo** [comment]
> > pushing seems to update the PR title with the latest commit title, I will change the title in the end after I've fixed the visual snapshots diff
> 
> It doesn't, but it's fine to update at the end.

Hm, my bad then. I will double check in the end though

**Comment by mivaylo** [comment]
@kevinbuhmann , you are right, pushing was not affecting title, I've opened it with wrong value in the beginning...

File: projects/angular/src/layout/tabs/_tabs.clarity.scss [hunk 1]
```diff
@@ -92,54 +81,38 @@ button.nav-link {
     box-shadow: none;
     flex-direction: column;
     align-items: stretch;
-    margin-right: variables.$clr_baselineRem_1;
+    margin-right: #{tokens.$cds-global-space-9};
```
Comment (valentin-mladenov) [inline projects/angular/src/layout/tabs/_tabs.clarity.scss:84]: Remove curly brackets.

```suggestion
    margin-right: tokens.$cds-global-space-9;
```

File: projects/angular/src/layout/tabs/_tabs.clarity.scss [hunk 2]
```diff
@@ -92,54 +81,38 @@ button.nav-link {
...
     overflow: auto;
     flex-shrink: 0;
-    padding: variables.$clr_baselineRem_4px;
-    width: variables.$clr_baselineRem_10;
-    min-width: variables.$clr_baselineRem_2;
+    padding: #{tokens.$cds-global-space-3};
+    width: calc(10 * #{tokens.$cds-global-space-9});
+    min-width: #{tokens.$cds-global-space-12};
```
Comment (valentin-mladenov) [inline projects/angular/src/layout/tabs/_tabs.clarity.scss:89]: Remove curly brackets.
```suggestion
    padding: tokens.$cds-global-space-3;
    width: calc(10 * #{tokens.$cds-global-space-9});
    min-width: tokens.$cds-global-space-12;
```

File: projects/angular/src/layout/tabs/_tabs.clarity.scss [hunk 3]
```diff
@@ -92,54 +81,38 @@ button.nav-link {
...
     align-items: stretch;
-    margin-right: variables.$clr_baselineRem_1;
+    margin-right: #{tokens.$cds-global-space-9};
     overflow: auto;
     flex-shrink: 0;
-    padding: variables.$clr_baselineRem_4px;
-    width: variables.$clr_baselineRem_10;
-    min-width: variables.$clr_baselineRem_2;
+    padding: #{tokens.$cds-global-space-3};
+    width: calc(10 * #{tokens.$cds-global-space-9});
```
Comment (valentin-mladenov) [inline projects/angular/src/layout/tabs/_tabs.clarity.scss:88]: Use `baselinePx` function for calculation to be more obvious, but is fine as it is, too.

```suggestion
    width: mixins.baselinePx(240);
```

**Review by valentin-mladenov — COMMENTED** [review]
Check my comments

**Review by valentin-mladenov — APPROVED** [review]

**Pull request merged**